common/djangoapps/student/tests/test_create_account.py

```diff
@@ -98,6 +99,13 @@ def test_extauth_bypass_sending_activation_email_without_bypass(self):
         """
         self.base_extauth_bypass_sending_activation_email(False)
 
+    @mock.patch.dict("student.models.settings.FEATURES", {"ENABLE_DISCUSSION_EMAIL_DIGEST": True})
+    def test_discussions_email_digest_pref(self):
+        response = self.client.post(self.url, self.params)
+        self.assertEqual(response.status_code, 200)
+        user = User.objects.get(username=self.username)
+        self.assertIsNotNone(UserPreference.get_preference(user, NOTIFICATION_PREF_KEY))
+
```

Comment:
I would suggest also testing the case of ENABLE_DISCUSSION_EMAIL_DIGEST = False (unless that is already tested elsewhere), to make sure that UserPreference.get_preference(user, NOTIFICATION_PREF_KEY) is None in that case.

I also agree that "NOTIFICATION_PREF_KEY" seems like a generic thing, but this feature is specifically about forum digests. Perhaps this is something that can be sorted out in the notifications work that @chrisndodge is doing?